ext/standard/tests/math/bindec_basic_64bit.phpt

```diff
@@ -29,49 +29,31 @@ $values = array(111000111,
 				null);
 
 for ($i = 0; $i < count($values); $i++) {
-	$res = bindec($values[$i]);
+    try {
+        $res = bindec($values[$i]);
+    } catch (InvalidArgumentException $e) {
+        $res = 'INVALID';
```

Comment:
It would be better to do like most other tests i.e.
```php
try {
    var_dump(bindec($values[$i]));
} catch (\WhatEverExceptionChosen $e) {
   echo $e->getMessage() . \PHP_EOL;
}
```